### PR TITLE
Configure username, email, password in processmaker:install

### LINKED ISSUE #2553: Installer User Setup
When running the initial installer, give the user options to configure:

username, email, password

Can we validate with the commands?

### PR DESCRIPTION
**Pull request by caleeli**
Resolves #2553 

This PR includes:

- Add options to configure username
- Add options to configure email
- Add options to configure password
- Includes validation of the fields
- Includes confirmation of password

image

### CODE REVIEW

File: ProcessMaker/Console/Commands/Install.php [hunk 1]
```diff
@@ -193,6 +197,56 @@ public function handle()
...
     }
 
+    /**
+     * Configure the username, email, password of the admin user
+     *
+     */
+    private function fetchUserInformation()
+    {
+        do {
+            $username = $this->anticipate(__('Enter your username'), ['admin'], 'admin');
```
Comment (marcoAntonioNina) [inline ProcessMaker/Console/Commands/Install.php]: missing labels

Comment (velkymx) [inline ProcessMaker/Console/Commands/Install.php]: Remove translations. At this point there isn't any languages installed.

File: ProcessMaker/Console/Commands/Install.php [hunk 2]
```diff
@@ -193,6 +197,56 @@ public function handle()
...
+            $validator = $this->validateField('password', $password, ['required', 'sometimes', 'min:6']);
+        } while($validator->fails());
+        do {
+            $confirmPassword = $this->secret(__('Confirm your password'));
+            $match = $password === $confirmPassword;
+            if (!$match) {
+                $this->error(__('Your password/confirm password fields do not match'));
+            }
+        } while(!$match);
+        putenv ("INSTALLER_ADMIN_USERNAME=$username");
```
Comment (velkymx) [inline ProcessMaker/Console/Commands/Install.php]: While I like this, it's a security issue. Remove the write to ENV and just save to DB

File: ProcessMaker/Console/Commands/Install.php (hunk 1 above)
Comment (caleeli) [reply ProcessMaker/Console/Commands/Install.php]: Translations removed

Comment (caleeli) [reply ProcessMaker/Console/Commands/Install.php]: Translations removed

**Comment by caleeli** [comment]
@velkymx, @marcoAntonioNina The translations where removed, and removed from env the admin properties for installation.

**Pull request merged**